src/components/template/Card.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+const Card = ({img,productName, productOwner, noOfRatings, price  }) => {
+    return (
```

Comment:
suggestion - you can also add an outOfStock prop which will store a boolean value & based on that you can change the css.

reference - https://neog-lms.vercel.app/docs/week-five/05-week-five-exercises#ex13-out-of-stock-grey